sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -124,7 +148,13 @@ public async Task GetTokenWithNullScopesShouldThrow()
         [Test]
         [TestCase("chat", TestName = "CreateUserAndTokenWithChatScope")]
         [TestCase("voip", TestName = "CreateUserAndTokenWithVoIPScope")]
-        [TestCase("chat", "voip", TestName = "CreateUserAndTokenWithMultipleScopes")]
+        [TestCase("chat.join", TestName = "CreateUserAndTokenWithChatJoinScope")]
```

Comment:
is it possible to use `CommunicationTokenScope.ChatJoin` instead of  providing the scope values?

Reply:
I tried but it didn't worked, and probably why earlier also we were passing arguments as string.